BraintreeCore/src/main/java/com/braintreepayments/api/ShopperInsightsRequest.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+/**
+ * Data class representing a request for shopper insights.
+ *
+ * Note: **This feature is in beta. It's public API may change in future releases.**
+ */
+data class ShopperInsightsRequest(
+    val email: String?,
```

Comment:
@sarahkoop we set these as `var`s, right?

Reply:
Yes 👍